shared/folders/dumb.js

```diff
@@ -213,31 +213,78 @@ function genFiles (offsetNumber: number, fileCount: number, isPrivate: boolean):
       path: 'pics/',
       modifiedMarker: false,
       fileIcon: 'logo-128',
-      onClick: () => console.log('onClick:file')
+      onClick: () => console.log('onClick:file', wordGen(i))
```

Comment:
i think we need some examples of really long filenames and things like this. my main folders list dumb component has really long lists of varying lengths so we can see how wrapping works